src/pages/Community/Community.js

```diff
@@ -1,6 +1,848 @@
 import styled from 'styled-components';
+import Slider from 'react-slick';
+import API from '../../conifg';
+import { ProductCard } from '../../components/ProductCard/ProductCard';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
 
 export function Community() {
-  return <div>Community</div>;
+  const navigate = useNavigate();
+
+  const settings = {
+    dots: false,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    autoplay: true,
+    autoplaySpeed: 4000,
+  };
+  const settings2 = {
+    dots: false,
+    infinite: false,
+    speed: 500,
+    slidesToShow: 10,
+    slidesToScroll: 5,
+    autoplay: false,
+  };
+
+  const [communityCarousel, setCommunityCarousel] = useState([]);
+  const [categoryCarousel, setCategoryCarousel] = useState([]);
+  const [communityNavigation, setCommunityNavigation] = useState([]);
+  const [communityHomeStory, setCommunityHomeStory] = useState([]);
+  const [communityHotPhoto, setCommunityHotPhoto] = useState([]);
+  const [productList, setProductList] = useState([]);
+  const [selectedCategoryProduct, setSelectedCategoryProduct] = useState([]);
+  const [category, setCategory] = useState([
+    { id: 1, title: '전체', select: true },
+    { id: 2, title: '가구', select: false },
+    { id: 3, title: '패브릭', select: false },
+    { id: 4, title: '조명', select: false },
+    { id: 5, title: '가전', select: false },
+    { id: 6, title: '주방용품', select: false },
+    { id: 7, title: '데코·식물', select: false },
+    { id: 8, title: '수납·정리', select: false },
+    { id: 9, title: '생활용품', select: false },
+    { id: 10, title: '생필품', select: false },
+    { id: 11, title: '공구·DIY', select: false },
+    { id: 12, title: '인테리어시공', select: false },
+    { id: 13, title: '반려동물', select: false },
+    { id: 14, title: '캠핑용품', select: false },
+    { id: 15, title: '실내운동', select: false },
+    { id: 16, title: '유아·아동', select: false },
+    { id: 17, title: '렌탈', select: false },
+    { id: 18, title: '식품', select: false },
+  ]);
+
+  useEffect(() => {
+    fetch(API.allProduct)
+      .then(res => res.json())
+      .then(data => {
+        setProductList(data.results);
+      });
+
+    fetch('/data/Carousel.json')
+      .then(res => res.json())
+      .then(data => {
+        setCommunityCarousel(data);
+      });
+
+    fetch('/data/CategoryCarousel.json')
+      .then(res => res.json())
+      .then(data => {
+        setCategoryCarousel(data);
+      });
+
+    fetch('/data/CommunityNav.json')
+      .then(res => res.json())
+      .then(data => {
+        setCommunityNavigation(data);
+      });
+
+    fetch('/data/CommunityHomeStory.json')
+      .then(res => res.json())
+      .then(data => {
+        setCommunityHomeStory(data);
+      });
+
+    fetch('/data/CommunityHotPhoto.json')
+      .then(res => res.json())
+      .then(data => {
+        setCommunityHotPhoto(data);
+      });
+
+    fetch(`${API.allProduct}?first_category=1`)
```

Comment:
1이라는 숫자가 어떤 의미를 가지는지 명확하게 변수화하면 좋을 것 같습니다.
```suggestion
    fetch(`${API.allProduct}?first_category=${ALL_CATEGORY_ID}`)
```

Reply:
수정하였습니다.